packages/components/organisms/f-checkout/src/components/AgeVerification.vue

```diff
@@ -0,0 +1,160 @@
+<template>
+    <card
+        is-rounded
+        is-page-content-wrapper
+        card-heading-position="center"
+        has-outline
+        :card-heading="$t(`ageVerification.heading`)"
+        data-test-id="checkout-age-verification-component">
+        <p
+            :class="$style['c-checkout-ageVerification-description']"
+            data-test-id="checkout-age-verification-description">
+            {{ $t(`ageVerification.description`) }}
+        </p>
+
+        <div>
+            <form-field
```

Comment:
Are these form field wrapped in a `form` element at any point? (hard to tell when split into components.

If not, should likely change the outer `div` to a `form` ideally so that it's better semantically.

Not sure on the requirements for this form, but would it be beneficial to have this submit to an endpoint? Just wondering if it's fully client-side validated, how easy it would be for people to get around it potentially (but may not even be an issue if that's accepted risk from product).

Reply:
I'll change it to a form, thanks.

The button will make a call to updateCheckout which should allow it to get properly validated.